cylc/flow/rundb.py

```diff
@@ -749,6 +749,17 @@ def select_submit_nums(self, name, point):
             ret[submit_num] = (flow_wait == 1, deserialise(flow_nums_str))
         return ret
 
+    def select_latest_flow_nums(self):
+        """Return a list of the most recent previous flow numbers."""
+        # Ignore bandit false positive: B608: hardcoded_sql_expressions
+        # Not an injection, simply putting the table name in the SQL query
+        # expression as a string constant local to this module.
+        stmt = (  # nosec
+            r"SELECT flow_nums, MAX(time_created) FROM %(name)s"
+        ) % {"name": self.TABLE_TASK_STATES}
```

Comment:
There's a now a pattern for these `nosec` comments in the file.

```suggestion
        stmt = rf'''
            SELECT flow_nums, MAX(time_created) FROM {self.TABLE_TASK_STATES}
        ''' # nosec (table name is code constant)
```